src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +59,64 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void filterTextFieldColor_validFilter_validFilterStyleApplied(){
+        testValidFilterStyleApplied("is:open");
+        testValidFilterStyleApplied("-is:open");
+        testValidFilterStyleApplied("hello");
+        testValidFilterStyleApplied("is:open & has:assignee");
+        testValidFilterStyleApplied("(is:closed)");
+    }
+
+    @Test
+    public void filterTextFieldColor_invalidFilter_invalidFilterStyleApplied(){
+
+        // Tests if parse errors applies invalid filter style to text field
+        testInvalidFilterStyleApplied("is:-open");
+        testInvalidFilterStyleApplied("--");
+        testInvalidFilterStyleApplied("is:-open & has:assignee");
```

Comment:
Hmm, think I pointed this out last time. Does this cover anything that the first one doesn't? If not it shouldn't be needed.

We're only testing that a parse error affects the field, so we don't really need that many examples.